tools/AutoTuner/test/smoke_test_algo_eval.py

```diff
@@ -60,33 +53,20 @@ def test_algo_eval(self):
             self.assertTrue(successful)
 
 
+class ASAP7AlgoEvalSmokeTest(BaseAlgoEvalSmokeTest):
+    platform = "asap7"
```

Comment:
noticed the removal of `make_base()` and `test_*()`. does the tests still run?

i am aware that the autotuner tests are disabled now, but just wanted to double check if tests can still trigger

Reply:
Yes, tests still run, they were triggered in some previous CIs, eg. [for GCD](https://jenkins.openroad.tools/job/OpenROAD-flow-scripts-Public/job/PR-2487-merge/1/pipeline-console/?start-byte=572748&selected-node=1061#log-910). I've just reduced code duplication by propagate these methods with inheritance